Solutions/Bitglass/Data Connectors/Bitglass_API_FunctionApp.json

```diff
@@ -0,0 +1,121 @@
+			}
+		]
+	},
+	"instructionSteps": [{
+			"title": "",
+			"description": ">**NOTE:** This connector uses Azure Functions to connect to the Azure Blob Storage API to pull logs into Azure Sentinel. This might result in additional costs for data ingestion and for storing data in Azure Blob Storage costs. Check the [Azure Functions pricing page](https://azure.microsoft.com/pricing/details/functions/) and [Azure Blob Storage pricing page](https://azure.microsoft.com/pricing/details/storage/blobs/) for details."
+		},
+		{
+			"title": "",
+			"description": ">**(Optional Step)** Securely store workspace and API authorization key(s) or token(s) in Azure Key Vault. Azure Key Vault provides a secure mechanism to store and retrieve key values. [Follow these instructions](https://docs.microsoft.com/azure/app-service/app-service-key-vault-references) to use Azure Key Vault with an Azure Function App."
+		},
+		{
+			"description": ">**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected [**Bitglass**](https://aka.ms/sentinel-Bitglass-parser) which is deployed with the Azure Sentinel Solution."
+		},
+		{
+			"title": "",
+			"description": "**STEP 1 - Configuration steps for the Bitglass Log Retrieval API**\n\n Follow the instructions to obtain the credentials.\n\n1. Obtain the **BitglassToken** and **BitglassServiceURL** using the [documentation]().\n2. Save credentials for using in the data connector."
+		},
+		{
+			"title": "",
+			"description": "**STEP 2 - Choose ONE from the following two deployment options to deploy the connector and the associated Azure Function**\n\n>**IMPORTANT:** Before deploying the Bitglass data connector, have the Workspace ID and Workspace Primary Key (can be copied from the following).",
+			"instructions": [{
+					"parameters": {
+						"fillWith": [
+							"WorkspaceId"
+						],
+						"label": "Workspace ID"
+					},
+					"type": "CopyableLabel"
+				},
+				{
+					"parameters": {
+						"fillWith": [
+							"PrimaryKey"
+						],
+						"label": "Primary Key"
+					},
+					"type": "CopyableLabel"
+				}
+			]
+		},
+		{
+			"title": "Option 1 - Azure Resource Manager (ARM) Template",
+			"description": "Use this method for automated deployment of the Bitglass data connector using an ARM Tempate.\n\n1. Click the **Deploy to Azure** button below. \n\n\t[![Deploy To Azure](https://aka.ms/deploytoazurebutton)](https://aka.ms/sentinel-BitglassAPI-azuredeploy)\n2. Select the preferred **Subscription**, **Resource Group** and **Location**. \n> **NOTE:** Within the same resource group, you can't mix Windows and Linux apps in the same region. Select existing resource group without Windows apps in it or create new resource group.\n3. Enter the **BitglassToken**, **BitglassServiceURL**  and deploy. \n4. Mark the checkbox labeled **I agree to the terms and conditions stated above**. \n5. Click **Purchase** to deploy."
```

Comment:
Please update the azure deploy link as **https://aka.ms/sentinel-bitglass-azuredeploy**

Reply:
Typo Tempate ->Template

Reply:
done